Microsoft.Toolkit.Graph/Providers/MockProvider.cs

```diff
@@ -42,22 +44,15 @@ private set
 
         /// <inheritdoc/>
         public GraphServiceClient Graph => new GraphServiceClient(
-            "https://proxy.apisandbox.msdn.microsoft.com/svc?url=" + HttpUtility.HtmlEncode("https://graph.microsoft.com/beta/"),
-            new DelegateAuthenticationProvider((requestMessage) =>
-            {
-                //// Temporary Workaround for https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/issues/59
-                //// ------------------------
-                var requestUri = requestMessage.RequestUri.ToString();
-                var index = requestUri.IndexOf("&");
-                if (index >= 0)
-                {
-                    requestMessage.RequestUri = new Uri(requestUri.Remove(index, 1).Insert(index, "?"));
-                }
-
-                //// End Workaround
-
-                return this.AuthenticateRequestAsync(requestMessage);
-            }));
+                        new DelegateAuthenticationProvider((requestMessage) =>
+                    {
+                        var requestUri = requestMessage.RequestUri.ToString();
+
+                        // Prepend Proxy Service URI to our request
+                        requestMessage.RequestUri = new Uri(GRAPH_PROXY_URL + HttpUtility.UrlEncode(requestUri));
```

Comment:
```suggestion
                        requestMessage.RequestUri = new Uri(GRAPH_PROXY_URL + Uri.EscapeDataString(requestUri));
```

Looks like we can use `Uri.EscapeDataString` here instead which reduces one more reference (System.Web) and is more compatible with the Uno build.